### DISCUSSION TITLE
Issue With Miner  Whitnesses in Hotspot but not on Chain Challenges appear as Whitness

### DISCUSSION THREAD

**Discussion by Exzellenz** [helium/miner#1609] [Issues]
Hello everybody, i am fighting with a new  Hotspot since a few Weeks.
Miner Version 2022.04.27.0_GA on a Browan MerryIot
This Hotspot Does Challenges but no Credited Beacons nor Credited Whitnesses. 
I did the usual Stuff like Multiple deletions of the Blockchain DB new Sd card and so on. 
I even changed the locations with one of my Working miners . Where the other miner was working great in that enviroment. 
This  one never had avalid whitness altough it is showing them  as sucessfully in the logs. 
It looks to me like there is something wrong with the keys during the initial setup .
i traced this transaction down:
`2022-05-06 05:57:42.688 6 [info] <0.25776.0>@miner_onion_server:decrypt:{372,13} could not decrypt packet received via radio: treating as a witness

2022-05-06 05:57:42.689 6 [info] <0.17855.2>@miner_onion_server:send_witness:{188,13} sending witness at RSSI: -115, Frequency: 868.3, SNR: 6.0

2022-05-06 05:57:43.877 6 [info] <0.17855.2>@miner_onion_server:send_witness:{251,37} successfully sent witness to challenger "/p2p/116wyqUXHsqAXrDH9yXBfMNbtDShKv4TNimjQ95hig2H52mT8SQ" with RSSI: -115, Frequency: 868.3, SNR: 6.0`

when i check this in the explorer This was a Challenge for the hotspot  Original Mango Shetland and not a  whitness.
So could someone tell me why it was not able to decrypt it? i would assume its a problem with the private key 
is there any way to generate a new key pair  ? 
Any help appreciated

**Comment by meowshka** [comment]
@Exzellenz let's see how your hotspot performs after the update to Light Hotspots, give it a few days.
Btw, we're moving all the issues/questions to the Discussions board, so your original report will be closed.
